### DISCUSSION TITLE
SEO 2.2 is out!

### DISCUSSION THREAD

**Discussion by h-enk** [gethyas/hyas#469] [Announcements]
SEO 2.2 is here! This release includes improved favicons generation, several bug fixes, and a new documentation website.

Highlights include:

- **Improved favicons generation**. Add a `png `favicon file (and optionally an `svg `favicon file) to your `assets `directory, and Hyas SEO generates all necessary files and meta tags.
- **New documentation website**. To better explain the possibilities of the Hyas SEO integration, we launched a new documentation website: [seo.gethyas.com](https://seo.gethyas.com/).

Read more in the [blog post announcement](https://gethyas.com/blog/seo-2-2/)

**Comment by huntsyea** [comment]
Way to go guys! Super pumped for this!
